### PR TITLE
Cleanup calculate.tpl to use standard CRM.$ style code rather than cj

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
This cleans up the code style of this javascript code to not be polluting the global name space and to use CRM.$ rather than cj

Before
----------------------------------------
Legacy code style

After
----------------------------------------
Current code style

ping @demeritcowboy @colemanw

### CODE REVIEW

File: templates/CRM/Price/Form/Calculate.tpl [hunk 1]
```diff
@@ -27,170 +27,170 @@
...
+      $('table#pricelabel').removeClass('disabled');
+    }
+
+    if (typeof skipPaymentMethod == 'function') {
+      // Advice to anyone who, like me, feels hatred towards this if construct ... if you remove the if you
+      // get an error on participant 2 of a event that requires approval & permits multiple registrants.
+      skipPaymentMethod();
+    }
+  }
+})(CRM.$);
```
Comment (mattwire) [inline templates/CRM/Price/Form/Calculate.tpl:194]:
```suggestion

  /**
   * Historically this function was on the window object. Now refactored to be within CRM.$
   * But quite a few extensions rely on calculateTotalFee() existing to get the total amount on the form.
   * @deprecated
   * @returns {number}
   */
  window.calculateTotalFee = function() {
      var totalFee = 0;
      $("#priceset [price]").each(function () {
        totalFee = totalFee + $(this).data('line_raw_total');
      });
      return totalFee;
  };

})(CRM.$);
```

**Comment by colemanw** [comment]
I think we should merge this tomorrow after 5.49 is branched. Any objections @mattwire?

**Comment by mattwire** [comment]
Well I released mjwshared (Payment Shared) 1.2.3 so it won't break any payment processors using that (Stripe etc).

But without my suggested change we will break various extensions: tsys, percentagepricefield, extra_fee, omnipay_paypal and maybe others and we are dealing with payments and amounts so I do not think it should be merged as-is because the consequences are severe (maybe wrong amounts being charged).  My suggestion is adding my suggestion to this PR and merging that until other extensions have had a chance to catch up.

**Comment by colemanw** [comment]
Ok. @seamuslee001 can you make that adjustment please?

**Comment by mattwire** [comment]
@seamuslee001 @colemanw I've just found out that both percentagepricesetfield and extrafee extensions replace the `calculateTotalFee()` function with their own. Any javascript-based processor needs to be able to read that function to get the right/adjusted amount. So my suggestion for changes still applies (but the comment should specifically reference extrafee/percentagepricesetfield) and we need to keep this function at the window scope until we have a suitable generic alternative in place (such as I do in mjwshared with `CRM.payment.getTotalAmount()`)

**Comment by eileenmcnaughton** [comment]
@seamuslee001 Am closing since the last ping was over 6 months ago - if you do want to open a new PR against this branch then at least it won't be on page 7 any more :-)

**Pull request closed without merging**